ebusd/CHANGELOG.md

```
@@ -1,21 +1,26 @@
 <!-- https://developers.home-assistant.io/docs/add-ons/presentation#keeping-a-changelog -->
-## version: 23.2.6
+
+<!-- append new version here -->
+## 4.4.4
```

Markdownlint

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)